llvm/lib/Passes/PassBuilder.cpp

```diff
@@ -828,11 +828,34 @@ parseLowerAllowCheckPassOptions(StringRef Params) {
     StringRef ParamName;
     std::tie(ParamName, Params) = Params.split(';');
 
-    return make_error<StringError>(
-        formatv("invalid LowerAllowCheck pass parameter '{0}' ", ParamName)
-            .str(),
-        inconvertibleErrorCode());
+    // cutoffs=990000|990000|0|...100000
+    if (ParamName.starts_with("cutoffs=")) {
+      StringRef CutoffsStr;
+      std::tie(ParamName, CutoffsStr) = ParamName.split('=');
+
+      while (CutoffsStr != "") {
+        StringRef firstCutoffStr;
+        std::tie(firstCutoffStr, CutoffsStr) = CutoffsStr.split('|');
```

Comment:
Can this be `,`?

Reply:
The problem is opt uses commas to separate passes, so `opt -passes=msan<cutoffs=1,2,3>,someOtherPass` gets misparsed into something like `msan<cutoffs=1`, `2`, `3>,`, `someOtherPass`.